packages/test/test-end-to-end-tests/src/test/data-virtualization/groupIdInSummary.spec.ts

```diff
@@ -82,21 +90,67 @@ describeCompat("Create data store with group id", "NoCompat", (getTestObjectProv
 		groupId: string | undefined,
 		message: string,
 	) => {
-		assert(snapshotTree.omitted, message);
```

Comment:
I don't think `assertOmittedTree` is valuable anymore, it was built for an older system. That or it needs to indicate that the blobs are missing/run `assertOmittedblobContents`.

Reply:
Read my other comment,